lib/classes/Variables.js

```diff
@@ -102,6 +102,10 @@ class Variables {
       const requiredConfigs = [
         _.assign({ name: 'region' }, provider.getRegionSourceValue()),
         _.assign({ name: 'stage' }, provider.getStageSourceValue()),
+        _.assign({ name: 'profile' }, {
+          value: this.service.provider.profile,
+          path: 'serverless.service.provider.profile'
```

Comment:
And test will be green!

```suggestion
          path: 'serverless.service.provider.profile',
```